docs/source/guide/tutorials/base/how_to_train/detection.rst

```diff
@@ -539,7 +539,7 @@ with OpenVINO™ PTQ.
             engine.optimize(checkpoint=ckpt_path)
 
 
-The optimization time highly relies on the hardware characteristics, for example on 1 NVIDIA GeForce RTX 3090 it took about 10 minutes.
+The optimization time highly relies on the hardware characteristics, for example on Intel(R) Core(TM) i9-11900 it took about 25 seconds.
```

Comment:
@sungmanc I updated the estimated time from 10 minutes to 25 seconds, is it correct?